Ports/RetroArch/package.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=RetroArch
+useconfigure="true"
+version="1.10.3"
+workdir="${port}-${version}"
+archive_hash="2af44294e55f5636262284d650cb5fff55c9070ac3a700d4fa55c1f152dcb3f2"
+files="https://github.com/libretro/${port}/archive/refs/tags/v${version}.tar.gz ${port}-${version}.tar.gz $archive_hash"
+auth_type=sha256
+depends=("freetype" "SDL2" "zlib")
+
+configopts=(
+    "--disable-builtinglslang"
+    "--disable-discord"
+    "--disable-glsl"
+    "--disable-glslang"
+    "--disable-opengl"
+    "--disable-slang"
+    "--disable-spirv_cross"
+    "--disable-update_cores"
+)
+
+launcher_name=RetroArch
+launcher_category=Games
+launcher_command=/usr/local/bin/retroarch
+icon_file=media/retroarch.ico
+
+function pre_configure() {
+    export CFLAGS="-I${SERENITY_INSTALL_ROOT}/usr/include/LibGL -I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2"
+}
+
+function post_configure() {
+    unset CFLAGS
+}
+
+post_install() {
+echo "==== Post installation instructions ===="
+echo "Please remember to use the Online updater"
+echo "to install cores info, assets, contents..."
+echo "before installing libretro cores from the port"
```

Comment:
Also can you write exactly what the user has to do to use the online updater? I assume you have to boot serenity and run RetroArch and choose the option from the menu? If that is the case I don't see why that would be necessary as the port build system does not access the disk image afaik.

Reply:
RetroArch needs some files to work correctly (core info files, assets, controller profiles ...). To "avoid" to package them I wrote this message so that we can get them from the online updater.

For now I think I will add only cores info files.  

When the GL driver is working, I will update the message for assets ( which will be used for xmb, ozone menu drivers).

When hid or sdl2 joypad driver is working I will update the message for controller profiles.

Reply:
[Update post install message](https://github.com/SerenityOS/serenity/pull/13855/commits/dc8f1ac667a033001c1f8bcd99341917664f4152)